src/Illuminate/Database/Console/PruneCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Illuminate\Database\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Events\Dispatcher;
+use Illuminate\Database\Eloquent\MassPrunable;
+use Illuminate\Database\Eloquent\Prunable;
+use Illuminate\Database\Events\ModelsPruned;
+use Illuminate\Support\Str;
+use Symfony\Component\Finder\Finder;
+
+class PruneCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'db:prune {--model=* : Class names of the models to be pruned}';
```

Comment:
I think the signature should be `models:prune`, since this is on the model level rather then the database level.